input/pages/index.md

```diff
@@ -1887,7 +1887,7 @@ If one or both of the operands is the empty collection, this operation returns a
 
 When comparing quantities for equality, the dimensions of each quantity must be the same, but not necessarily the unit. For example, units of `'cm'` and `'m'` can be compared, but units of `'cm2'` and `'cm'` cannot. The comparison will be made using the most granular unit of either input. Attempting to operate on quantities with invalid units will result in empty (`{ }`).
 
-For time-valued quantities, note that calendar durations and definite quantity durations above seconds are considered unequal:
+For time-valued quantities, note that calendar durations and definite quantity durations above days (and weeks) are considered un-comparable:
 
 ``` fhirpath
 1 year = 1 'a' // false
```

Comment:
fhirpath.js also returns empty.

Reply:
Is this the one that you're referring to?
(rather than the text that I updated?)
firely returns empty, fhirpath.js returns false, HAPI breaks.
Should I be updated the result here to empty also (which wasn't in the approved resolution)
```fhirpath
1 year = 1 'a'
```
:test_tube: [Test with FHIRPath-Lab](https://dev.fhirpath-lab.com/FhirPath?parameters=N4IgpgHgDgTmDO8CWB7AdiAXCAjAAgE8wBDGPAXj3wHJjqQAacNAcyTTCxADMALJGFGIAXXgDoAVvEYhhYGAFt2KADYoWBAMryAbvK69hwqPEwB6M-ACOatHwEBaGABYxxAF4BXOAHcwAI2Q5eDEOYTN7GBkdUiRifxUELABtAF0mOHgUbwBjMAApLIxsYAAdEEzsmDyAFQIoMHLMcoAFESQwNGFyhnKkABMm8shiBShE8oBfEEmgA)

Reply:
> fhirpath.js returns false,
Version 3.10.1 (https://hl7.github.io/fhirpath.js/) returns empty.

Reply:
Example also corrected.